app/controllers/helpers/grocery_lists_helper.rb

```diff
@@ -0,0 +1,5 @@
+module GroceryListsHelper
+  def list_params
```

Comment:
Strong params normally go in the controller. The helper normally contain helper methods for views to use.